### ISSUE TITLE
Getting Started Docs - snippet error

### ISSUE THREAD

**Issue by simonlsk** [open-telemetry/opentelemetry.io#644]
In the **Creating metric instruments** section the snippet includes the line:
```
meter := otel.Meter("ex.com/basic")
```
For which I get an unresolved reference. I believe the function `Meter` is in `global` and the correct line should be:
```
meter := global.Meter("ex.com/basic")
```

That waisted me about an hour trying to figure out what I was doing wrong.

Thank you!

**Comment by chalin** [comment]
I believe that this is reporting the same issue as https://github.com/open-telemetry/opentelemetry-go/issues/2087? It has been fixed (https://github.com/open-telemetry/opentelemetry-go/pull/2093), but the updated docs haven't been ported.

I'm going to mark this as a duplicate, but leave it open until the docs are updated in this repo.

/cc @Aneurysm9 @lastchiliarch

**Comment by chalin** [comment]
Closed by #690
